File: inst/htmlwidgets/hello.js
HTMLWidgets.widget({

  name: 'hello',

  type: 'output',

  initialize: function(el, width, height) {

    
    //alert('hello');
    



    return {
      // TODO: add instance fields as required
    };

  },

  renderValue: function(el, x, instance) {

    //el.innerText = x.message;
    el.innerText = 'hello...';
    el.setAttribute('style', 'background-color: red');
    
    alert('hello');
    
    
  },

  resize: function(el, width, height, instance) {
     
    alert('hello');
    
  }

});

File: R/hello.R
#' <Add Title>
#'
#' <Add Description>
#'
#' @import htmlwidgets
#'
#' @export
hello <- function(message = "Hello world!", width = NULL, height = NULL) {
  
  # forward options using x
  x = list(
    message = message
  )
  
  # create widget
  htmlwidgets::createWidget(
    name = 'hello',
    x,
    width = width,
    height = height,
    package = 'radarchart'
  )
}